src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1705,6 +1705,9 @@ private function registerSecurityCsrfConfiguration(array $config, ContainerBuild
     private function registerSerializerConfiguration(array $config, ContainerBuilder $container, PhpFileLoader $loader)
     {
         $loader->load('serializer.php');
+        if ($container->getParameter('kernel.debug')) {
+            $loader->load('serializer_debug.php');
```

Comment:
No need for a new file to load, let's remove in this class directly.

Reply:
Good point!